fdbclient/DatabaseContext.h

```diff
@@ -133,22 +133,26 @@ class DatabaseContext : public ReferenceCounted<DatabaseContext>, public FastAll
 
 	std::map< UID, StorageServerInfo* > server_interf;
 
-	Standalone<StringRef> dbId;
-
-	int64_t transactionReadVersions;
-	int64_t transactionLogicalReads;
-	int64_t transactionPhysicalReads;
-	int64_t transactionCommittedMutations;
-	int64_t transactionCommittedMutationBytes;
-	int64_t transactionsCommitStarted;
-	int64_t transactionsCommitCompleted;
-	int64_t transactionsTooOld;
-	int64_t transactionsFutureVersions;
-	int64_t transactionsNotCommitted;
-	int64_t transactionsMaybeCommitted;
-	int64_t transactionsResourceConstrained;
-	int64_t transactionsProcessBehind;
-	int64_t transactionWaitsForFullRecovery;
+	UID dbId;
+	bool internal;
```

Comment:
I feel it is better to add a comment behind `internal` to clarify: what request are considered as internal. 
The comment can be:
`// true if request is from FDB components, e.g., backup and restore.`

It can save time when someone reads the code one year later.